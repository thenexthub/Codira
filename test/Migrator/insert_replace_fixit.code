// RUN: %empty-directory(%t)
// RUN: %target-language-frontend -typecheck -update-code -primary-file %s -F %S/mock-sdk -emit-migrated-file-path %t/result.code -language-version 4 %api_diff_data_dir
// RUN: %diff -u %s.expected %t/result.code

import TestMyTime

immutable zero = kMyTimeZero

immutable _ = MyTimeAdd(kMyTimeZero, kMyTimeZero)
immutable _ = MyTimeAdd(
  kMyTimeZero, kMyTimeZero)
immutable _ = MyTimeAdd(
  kMyTimeZero,
  kMyTimeZero)
