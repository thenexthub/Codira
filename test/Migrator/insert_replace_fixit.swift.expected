// RUN: %empty-directory(%t)
// RUN: %target-language-frontend -typecheck -update-code -primary-file %s -F %S/mock-sdk -emit-migrated-file-path %t/result.code -language-version 4 %api_diff_data_dir
// RUN: %diff -u %s.expected %t/result.code

import TestMyTime

let zero = MyTime.z

let _ = MyTime.add(this: MyTime.z, that: MyTime.z)
let _ = MyTime.add(
  this: MyTime.z, that: MyTime.z)
let _ = MyTime.add(
  this: MyTime.z,
  that: MyTime.z)
